File: 30-days-of-code/Day03/Statements.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    Statements.py                                      :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/05 19:00:23 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/05 19:50:45 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

#!/bin/bash 

import math
import os
import random
import re
import sys

if __name__ == '__main__':
    N = int(input())
    if N % 2 == 1 :
        print("Weird")
    elif N >= 2 and N <= 5 and N % 2 == 0 :
        print("Not Weird")
    elif N >= 6 and N <= 20 and N % 2 == 0 :
        print("Weird")
    else
        print("Not Weird")

File: 30-days-of-code/Day03/Statements.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Statement.java                                     :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/25 16:24:33 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/25 16:24:42 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.Scanner;

public class Statements
{
	public static void main(String[] args)
	{
		Scanner scanInput = new Scanner(System.in);

		int n = scanInput.nextInt();

		ScanInput.close();

		if (n % 2 == 1)
			System.out.println("Weird");
		else if (n % 2 == 0 && n >= 2 && n <= 5)
			System.out.println("Not Weird");
		else if (n % 2 == 0 && n >= 6 && n <= 20)
			System.out.println("Weird");
		else
			System.out.println("Not weird");
	}
}
